manager/refresh.sh

```diff
@@ -30,7 +30,7 @@ eksctl utils write-kubeconfig --cluster=$CORTEX_CLUSTER_NAME --region=$CORTEX_RE
 out=$(kubectl get pods 2>&1 || true); if [[ "$out" == *"must be logged in to the server"* ]]; then echo "error: your aws iam user does not have access to this cluster; to grant access, see https://docs.cortex.dev/v/${CORTEX_VERSION_MINOR}/"; exit 1; fi
 
 kubectl get -n=default configmap cluster-config -o yaml >> cluster_configmap.yaml
-python refresh_cluster_config.py cluster_configmap.yaml tmp_cluster_config.yaml
+python manager/refresh_cluster_config.py cluster_configmap.yaml tmp_cluster_config.yaml
 
 kubectl -n=default create configmap 'cluster-config' \
```

Comment:
use the namespace env var here and on line 26 (it might be worth searching the manager folder for `default`)